lib/components/dev-badge/index.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
 const DevBadge = () => {
-  return <div className="dev-badge">DEV</div>;
+  return (
+    <div aria-hidden="true" className="dev-badge">
+      DEV
+    </div>
+  );
```

Comment:
while we're at it, can we have this return the value directly instead of opening up a function with return here?

```ts
const DevBadge = () => (
	<div aria-hidden="true" className="dev-badge">
		DEV
	</div>
);
```